interfaces/api/pipeline/IMapUpdatePipeline.h

```diff
@@ -46,6 +46,20 @@ class XPCF_CLIENTUUID("d9da863c-c9ff-4562-a3a2-329ac1f44008") XPCF_SERVERUUID("8
 	/// @brief IMapUpdatePipeline default destructor
 	virtual ~IMapUpdatePipeline() = default;
 
+    /// @brief Set the map to use for this Map Update pipeline instance
+    /// @param[in] uuid: the map UUID
```

Comment:
Don't use `:` in doc, just leave a space between the name and the description (here and elsewhere)

Reply:
Ok, will try to fix it in all interfaces